prow-job-kubestellar-kubeflex-compatibility.yaml

```diff
@@ -0,0 +1,177 @@
+# Prow job for KubeStellar/KubeFlex compatibility E2E testing
+# Place this in kubestellar/infra/prow/jobs/kubestellar/kubestellar-kubeflex-compatibility.yaml
+presubmits:
+  - name: pull-kubestellar-kubeflex-compatibility-e2e
+    decorate: true
+    optional: true
+    max_concurrency: 2
+    branches:
+      - main
+    trigger: "(?m)^/test compatibility-e2e$"
+    rerun_command: "/test compatibility-e2e"
+    run_if_changed: ".*"
+    context: prow-compatibility-e2e
+    cluster: oci-prow
+    timeout: 3h
+    spec:
+      containers:
+        - image: ghcr.io/kubestellar/infra/build:1.24.2-1
+          command:
+            - bash
+            - -c
+            - |
+              set -ex
+              ./scripts/e2e-compatibility-matrix.sh
```

Comment:
What is this tiny bit of bash scripting adding? Why not simply write the following?

```yaml
          command:
            - ./scripts/e2e-compatibility-matrix.sh
```